### ISSUE TITLE
SyntaxError, unrecognized expression =>

### ISSUE THREAD

**Issue by ahmadalfy** [drdrsh/grunt-expand-in-place#4]
I know that this error is probably due to failure to comprehend the arrow function. Perhaps you need to clarify the version of Node and NPM that are required for your task to run.

```
Loading "expand-in-place.js" tasks...ERROR
>> SyntaxError: path-to-code\node_modules\grunt-expand-in-place\tasks\expand-in-place.js:16
>>             'css': file => {
>>                         ^^
>> Unexpected token =>
```

NodeJs version : v0.12.2
NPM version : 2.7.4

**Comment by drdrsh** [comment]
Actually @saadshahd reported and fixed this issue, it is already included in package.json "engines" key should I set it somewhere else?

**Comment by ahmadalfy** [comment]
My engine version meet your requirements

```
    "node": ">= 0.8.0"
```

So you probably need a higher version. I am upgrading to the latest version of Node now and will report back if I am still having troubles.

**Comment by drdrsh** [comment]
I dunno, the file on the repository is 
https://github.com/drdrsh/grunt-expand-in-place/blob/master/package.json
And it clearly has node: ">= 4.0.0"

Maybe you need to update your local cache

**Comment by drdrsh** [comment]
It could be that I was careless and didn't increment the version number so you are getting the wrong revision. Will fix that

**Comment by ahmadalfy** [comment]
It's probably that because it's my first time to use your package ... Will close the issue now cause it's clearly that reason.

**Comment by ahmadalfy** [comment]
Perfect, updating my NodeJS to the latest version resolved it.

**Comment by drdrsh** [comment]
Actually I realized I didn't "release" the changes that Saad made, I am so careless  
Fixed in a40cb88713848936db345d56ea56fdebb01068c5 and published to npm
